### PR TITLE
Update the CSS coding guidelines

### PR DESCRIPTION
**Pull request by manzoorwanijk**
#### Proposed Changes

- Update the CSS coding guidelines as a part of #67467

### CODE REVIEW

**Comment by noahtallen** [comment]
Nice, thanks for this! Can we also update the rest of the file to match the new coding standards as well? We especially need to update the code snippets in the document.

**Comment by manzoorwanijk** [comment]
> Can we also update the rest of the file to match the new coding standards as well? We especially need to update the code snippets in the document.

Done. Thank you @noahtallen

File: docs/coding-guidelines/css.md [hunk 1]
```diff
@@ -275,9 +275,9 @@ selector {
 - DO use a single space before an opening brace
```
Comment (noahtallen) [inline docs/coding-guidelines/css.md]: we should update the line above this about spaces around parenthesis as well

**Review by noahtallen — APPROVED** [review]
other than my one comment, I think this is good to go!

File: docs/coding-guidelines/css.md (hunk 1 above)
Comment (manzoorwanijk) [reply docs/coding-guidelines/css.md]: I already updated that.

**Review by noahtallen — APPROVED** [review]
Oh, my bad! I must have been reviewing an out-of-date page.

**Pull request merged**